src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -69,6 +82,24 @@ export default function withAnnotations<P extends object>(
             this.state = { ...defaultState, activeAnnotationId };
         }
 
+        componentDidUpdate(
+            _prevProps: WithAnnotationsProps,
+            { error: prevError, hasAnnotatorInit: prevHasAnnotatorInit }: AnnotatorState,
+        ) {
+            const { error, hasAnnotatorInit } = this.state;
+
+            const hasAnnotatorChanged = prevHasAnnotatorInit !== hasAnnotatorInit;
+            const hasErrorChanged = prevError !== error;
+
+            if ((hasAnnotatorChanged || hasErrorChanged) && error && hasAnnotatorInit) {
+                this.escalateAnnotationsFetchError(error);
+
+                if ('origin' in error && error.origin === ORIGIN_ACTIVITY_SIDEBAR && this.annotator) {
+                    this.annotator.emit('annotations_visible_set', false);
```

Comment:
Should we instead tell preview to destroy the annotator and/or prevent it from creating it in the first place? Do we want the region toggle to show if the fetch call fails?